code/modules/antagonists/antag_org/antag_org_syndicate.dm

```diff
@@ -0,0 +1,84 @@
+/// Base syndicate org datum
+/datum/antag_org/syndicate
+	name = "Buggy Syndicate Corp, ahelp this please"
+	chaos_level = ORG_CHAOS_AVERAGE
+
+/datum/antag_org/syndicate/donk // Completely random objectives, default traitor
+	name = "Donk Co."
+	intro_desc = "You are a Donk Co. agent, sent here to advance Syndicate interests. \
+		Current client is anonymous. Standard rules of engagement apply. Get the job done, and get it done right."
+
+/datum/antag_org/syndicate/hawkmoon // Theft only
+	name = "Hawkmoon Acquisitions"
+	intro_desc = "You are an incursion specialist from the Hawkmoon Acquisitions Corporation, a merchandising firm using less-than-legal methods of product procurement. \
+		Grab the goods, keep it quiet, leave no trace. We were never here."
+	objectives = list(/datum/objective/steal)
+	chaos_level = ORG_CHAOS_MILD
+
+/datum/antag_org/syndicate/arc // Only targets on-station Cargo/Service/Genetics/Virologist
+	name = "Animal Rights Consortium"
+	intro_desc = "You are a member of the Animal Rights Consortium, here to violently protest the cruel treatment of animals by megacorporations like Nanotrasen. \
+		Teach them a lesson!"
+	objectives = list(/datum/objective/assassinateonce/arc)
+	chaos_level = ORG_CHAOS_MILD // Violent but never needs to permakill
+
+/datum/antag_org/syndicate/waffle // Assassination variants only
+	name = "Waffle Company"
+	intro_desc = "You are a contract killer under the employ of Waffle Co., a ruthless criminal entity that will go after any target, for the right price. \
+	Got a few new bounties on the docket, agent. Put 'em down however you see fit."
+	objectives = list(/datum/objective/assassinate, /datum/objective/assassinateonce, /datum/objective/maroon)
+
+/datum/antag_org/syndicate/cybersun // Mostly target Command/Security
+	name = "Cybersun Incorporated - The Inner Circle"
+	intro_desc = "You're an operative of Cybersun Incorporated's Inner Circle, an elite PMC and proxy arm of the company. \
+		Clean kills, clean thefts, clean getaway. Get it done, operative."
+	focus = 50 // Don't bully sec too hard
+	targeted_departments = list(DEPARTMENT_COMMAND, DEPARTMENT_SECURITY)
+	theft_targets = list(/datum/theft_objective/antique_laser_gun, /datum/theft_objective/nukedisc, /datum/theft_objective/hoslaser, \
```

Comment:
maybe consider converting this in a multiline list definition?

Reply:
Is there any specific pro to doing that? Or just a formatting thing

Reply:
Formatting thing

Reply:
I made a suggestion, and Steel pushed it in. We are good.